ttnn/cpp/ttnn/operations/data_movement/expand/device/kernels/reader_rm_expand.cpp

```diff
@@ -0,0 +1,60 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include <cstdint>
+
+#include "dataflow_api.h"
+#include "debug/dprint.h"
+#include "risc_attribs.h"
+#include "tensix.h"
```

Comment:
It seems that most operation's data movement kernels do not include the `tensix` header. Therefore, this might not be necessary.

Reply:
I've removed the headers